scripts/src/versioning.py

```diff
@@ -0,0 +1,77 @@
+import os
+import re
+from pathlib import Path
+from typing import List
+
+import yaml
+
+meta_info_name = "meta.yaml"
+
+
+def folder_name_to_option(name: str):
+    return name.replace("_", ".")
+
+
+def version_to_folder_name(version: tuple):
+    major = get_major(version)
+    return "v" + "_".join([str(x) for x in major])
+
+
+def version_to_setting_name(version: tuple):
```

Comment:
Wondered what this does based on the function name, but looks like it's not in use? If so: Remove?

Reply:
Not used, removed